src/communication.h

```diff
@@ -145,13 +145,16 @@ std::pair<Property, SimpleVariant> processCanMessage(const std::vector<std::uint
 
     const auto value{static_cast<std::uint16_t>((byte1 << 8U) | byte2)};
     const auto canId{static_cast<std::uint16_t>(((msg[0U] & 0xfc) << 3) | (msg[1U] & 0x3f))};
-    ESP_LOGI("Communication",
-             "Message received: Read/Write ID 0x%02x 0x%02x(0x%03x) for property %s (0x%04x) with raw value: %d",
-             msg[0U], msg[1U], canId, std::string(property.name).c_str(), property.id, value);
     if (isRequest(msg)) {
         ESP_LOGD("Communication", "Message is a request. Dropping it!");
+        ESP_LOGD("Communication",
+                 "Message received: Read/Write ID 0x%02x 0x%02x(0x%03x) for property %s (0x%04x) with raw value: %d",
+                 msg[0U], msg[1U], canId, std::string(property.name).c_str(), property.id, value);
         return {Property::kINDEX_NOT_FOUND, value};
     }
+    ESP_LOGI("Communication",
+             "Message received: Read/Write ID 0x%02x 0x%02x(0x%03x) for property %s (0x%04x) with raw value: %d",
+             msg[0U], msg[1U], canId, std::string(property.name).c_str(), property.id, value);
```

Comment:
Isn't it the same amount of log messages?

Reply:
notice the LOGD vs. LOGI 😉